cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/partitioned/PartitionedFileSetAdmin.java

```diff
@@ -36,35 +31,16 @@
 public class PartitionedFileSetAdmin extends CompositeDatasetAdmin {
   private final DatasetContext context;
   private final DatasetSpecification spec;
-  private final Provider<ExploreFacade> exploreFacadeProvider;
 
   public PartitionedFileSetAdmin(DatasetContext context, DatasetSpecification spec,
-                                 Provider<ExploreFacade> exploreProvider,
                                  Map<String, DatasetAdmin> embeddedAdmins) {
     super(embeddedAdmins);
     this.context = context;
     this.spec = spec;
-    this.exploreFacadeProvider = exploreProvider;
   }
 
   @Override
   public void truncate() throws IOException {
```

Comment:
No need to override just to call `super`

Reply:
Removed